docs/design/ovs-pipeline.md

```diff
@@ -788,11 +788,11 @@ Flow 1 is for case 1, matching packets received on the local Antrea gateway port
 addresses. There are some cases where the source IP of the packets through the local Antrea gateway port is not the local
 Antrea gateway IP address:
 
-- When Antrea is deployed with kube-proxy, and `AntreaProxy` is not enabled, packets from local Pods destined for Services
+- When Antrea is deployed with kube-proxy, and feature `AntreaProxy` is not enabled, packets from local Pods destined for Services
   will first go through the gateway port, get load-balanced by the kube-proxy data path (undergoes DNAT) then re-enter
   the OVS pipeline through the gateway port (through an "onlink" route, installed by Antrea, directing the DNAT'd packets
   to the gateway port), resulting in the source IP being that of a local Pod.
-- When Antrea is deployed without kube-proxy, and both `AntreaProxy` and `proxyAll` are enabled, packets from the external
+- When Antrea is deployed without kube-proxy, and both feature `AntreaProxy` and option `proxyAll` are enabled, packets from the external
```

Comment:
"both the `AntreaProxy` feature and option `proxyAll`"

Reply:
Done